apps/web/app/global-error.tsx

```
@@ -0,0 +1,31 @@
+'use client'; // Error boundaries must be Client Components
+//TODO 임시로 글로벌 에러바운더리 설정
+
+import { useEffect } from 'react';
+
+export default function Error({
+  error,
+  reset,
+}: {
+  error: Error & { digest?: string };
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)